quickwit/quickwit-config/src/storage_config.rs

```diff
@@ -157,13 +168,14 @@ pub enum StorageConfig {
     File(FileStorageConfig),
     Ram(RamStorageConfig),
     S3(S3StorageConfig),
+    Google(GoogleCloudStorageConfig),
 }
 
 impl StorageConfig {
     pub fn redact(&mut self) {
         match self {
             Self::Azure(azure_storage_config) => azure_storage_config.redact(),
-            Self::File(_) | Self::Ram(_) => {}
+            Self::File(_) | Self::Ram(_) | Self::Google(_) => {}
```

Comment:
I curious why `Self::Google` does not follow  `Self::S3` (and `Self::Azure`) logic with redact?